### ISSUE TITLE
Dead project ??

### ISSUE THREAD

**Issue by pedromorgan** [mapnik/OGCServer#38]
Dont see much activity and having problems running.. ???

is this a maintainer missing ??

**Comment by manelclos** [comment]
@pedromorgan what problem are you having?

**Comment by anibalpacheco** [comment]
@manelclos For example this:

```
File ".../ogcserver/common.py", line 543, in xmlhandler
    return Response(self.xmlmimetype, ElementTree.tostring(ogcexcetree, pretty_print=True), status_code=404)
 TypeError: tostring() got an unexpected keyword argument 'pretty_print'
```

I give up, I spent the whole day (more than 12 hours) trying to move a working instance of this project from an old server to a updated server, but nothing works. I've tried centos7 compiling everithing (mapnik, python-mapnik) on many branches, all errors,  ArchLinux with the AUR packages or compliling also, again errors trying many branches, and finally Ubuntu18.04 with the system packages, and gave me this pretty print error, I've tried to remove the usage of the parameter, then I receive a 404 in the getCapabilities without any log, using mod_wsgi. I got a 200 OK for Capabilities using "ogcserver mapfile.xml" but 404 to all my postgis layers the important ones, a 200 OK for an esri-shapefile layer but without styles.

**Comment by manelclos** [comment]
Yep, this pretty much needs an update. I haven't been using the project and I'm low on spare time.
